### PR TITLE
Gd 43 fix wearable nft example

### PR DESCRIPTION
**Pull request by Krayt78**

### CODE REVIEW

File: Assets/MirageSDK/Examples/Scripts/WearableNFTExample/WearableNFTExample.cs [hunk 1]
```diff
@@ -44,17 +43,14 @@ public async void RunExample()
...
 			await GetHat();
 			await ChangeHat(RedHatAddress);
 			await GetHat();
 		}
-		public async void MintNFTsCall()
-		{
-			//await MintItems();
-			await MintCharacter();
-		}
+
```
Comment (Vukiz) [inline Assets/MirageSDK/Examples/Scripts/WearableNFTExample/WearableNFTExample.cs:52]: Remove extra line

File: Assets/MirageSDK/Examples/Scripts/WearableNFTExample/WearableNFTExample.cs [hunk 2]
```diff
@@ -65,16 +61,16 @@ private async UniTask MintItems()
...
 			};
 			var itemsAmounts = new[]
 			{
 				1, 2, 3, 4, 5, 6
 			};
 
 			var receipt = await _gameItemContract.CallMethod(mintBatchMethodName,
-				new object[] { itemsToMint, itemsAmounts });
-			Debug.Log($"Game Items Minted. Receipts : {receipt}");
+				new object[] {"0x510c522ebCC6Eb376839E0CFf5D57bb2F422EB8b", itemsToMint, itemsAmounts, new byte[] { }});
```
Comment (Vukiz) [inline Assets/MirageSDK/Examples/Scripts/WearableNFTExample/WearableNFTExample.cs:72]: move hardcoded address to const

Comment (Vukiz) [reply Assets/MirageSDK/Examples/Scripts/WearableNFTExample/WearableNFTExample.cs:72]: also extract `new byte[] {}` into local variable

File: Assets/MirageSDK/Examples/Scripts/WearableNFTExample/WearableNFTExample.cs [hunk 3]
```diff
@@ -119,63 +105,108 @@ private async UniTask<BigInteger> GetBalance()
...
 		}
 
 		private void UpdateUILogs(string log)
 		{
-			_text.text = log;
+			_text.text += "\n" + log;
+			Debug.Log(log);
+		}
+
+		#region Button Calls
```
Comment (Vukiz) [inline Assets/MirageSDK/Examples/Scripts/WearableNFTExample/WearableNFTExample.cs:168]: region should be indented one tab less

**Review by Vukiz — CHANGES_REQUESTED** [review]

File: Assets/MirageSDK/Examples/Scripts/WearableNFTExample/WearableNFTExample.cs (hunk 3 above)
Comment (Krayt78) [reply Assets/MirageSDK/Examples/Scripts/WearableNFTExample/WearableNFTExample.cs:168]: i might have to reconfigure my rider as i used the reformat code & cleanup before the pull request. 
Could you send me your rider config ?

**Review by Vukiz — APPROVED** [review]

**Pull request merged**